packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.11.0+3
+
+* Disposing Camera Example app's focus mode row animation controller.
```

Comment:
You should be able to remove this. This doesn't require a version bump since it is only changing the example app: https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#version.

Reply:
This is the part of the example that's published, so we do version it.